server/test/unit/plugins/conduits/test_mixins.py

```diff
@@ -2,14 +2,73 @@
 
 from ... import base
 from pulp.devel import mock_plugins
-from pulp.plugins.conduits.mixins import ImporterScratchPadMixin, ImporterConduitException
+from pulp.plugins.conduits.mixins import (DistributorScratchPadMixin, DistributorConduitException,
+                                          ImporterScratchPadMixin, ImporterConduitException)
 from pulp.server.db.model.repository import Repo
 from pulp.server.managers.repo.cud import RepoManager
+from pulp.server.managers.repo.distributor import RepoDistributorManager
 from pulp.server.managers.repo.importer import RepoImporterManager
 import pulp.plugins.types.database as types_database
 import pulp.server.managers.factory as manager_factory
 
 
+class DistributorScratchpadMixinTests(base.PulpServerTests):
```

Comment:
@rbarlow Do you think future versions of Pulp will want class docblocks and function docblocks for test code too? Not necessary for this commit, but I am asking in general.